doc/classes/DisplayServer.xml

```diff
@@ -1980,21 +1996,24 @@
 		<constant name="MOUSE_MODE_MAX" value="5" enum="MouseMode">
 			Max value of the [enum MouseMode].
 		</constant>
+		<constant name="INVALID_SCREEN" value="-1">
+			The ID that refers to a nonexistent screen. This is returned by some [DisplayServer] methods if no screen matches the requested result.
+		</constant>
 		<constant name="SCREEN_WITH_MOUSE_FOCUS" value="-4">
-			Represents the screen containing the mouse pointer.
-			[b]Note:[/b] On Linux (Wayland), this constant always represents the screen at index [code]0[/code].
+			Represents the screen containing the mouse pointer. Can be used as parameter for some [DisplayServer] methods.
```

Comment:
Suggestion applies for the others as well.

This conveniently fits into [what I remarked just a few days ago](https://chat.godotengine.org/channel/documentation?msg=ftRe6KQ3GQfbfCWCz).
```suggestion
			Represents the screen containing the mouse pointer. Can be used as an argument for some [DisplayServer] methods.
```
Although, uh... Yeah, I mean. They're constants that... can be used. They are documented as such in this very PR. I don't think there's any need to say this for each individual constant.
```suggestion
			Represents the screen containing the mouse pointer.
```